convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,431 @@
+        )
+        LOGGER.debug("Tokens and annotations uploaded")
+
+    def download_label_studio_from_s3(
+        self,
+        s3_input_annotation: S3Path,
+    ) -> LabelStudioModel:
+        with tempfile.TemporaryDirectory() as tmp_dirname:
+            input_file = (
+                Path(tmp_dirname) / Path(s3_input_annotation.path).name
+            )
+
+            try:
+
+                self.s3_client.download_file(
+                    s3_input_annotation.bucket,
+                    s3_input_annotation.path,
+                    str(input_file),
+                )
+            except ClientError as e:
+                LOGGER.exception(
+                    "Cannot download file from S3: bucket %s path %s",
+                    s3_input_annotation.bucket,
+                    s3_input_annotation.path,
+                )
+                raise HTTPException(
+                    status_code=404,
+                    detail="Error during file fetching: file not found",
+                ) from e
+
+            result = LabelStudioModel.parse_file(input_file)
+            # result = LabelStudioModel.parse_file(Path("tests/test_label_studio/test_data/label_studio_format.json"))
+            return result
+
+    def get_output_tokens_path(self, file_id_in_assets: int) -> str:
+        return (
+            f"files/{file_id_in_assets}/ocr/{self.converted_tokens_filename}"
+        )
+
+    def get_output_pdf_path(self, file_id_in_assets: int) -> str:
+        return f"files/{file_id_in_assets}/{file_id_in_assets}.pdf"
+
+    def get_output_annotations_path(
+        self, importjob_id_created: int, file_id_in_assets: int
+    ) -> str:
+        return f"annotation/{importjob_id_created}/{file_id_in_assets}/{self.converted_annotations_filename}"
+
+    def make_upload_file_request_to_assets(self, pdf_path):
+        upload_file_to_assets_url = f"{settings.assets_service_url}"
+        files = [
+            (
+                "files",
+                (pdf_path.name, open(pdf_path, "rb"), "application/pdf"),
+            ),
+        ]
+        try:
+            request_to_post_assets = requests.post(
+                url=upload_file_to_assets_url,
+                headers=self.request_headers,
+                files=files,
+            )
+            request_to_post_assets.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            LOGGER.exception(
+                "Failed request to 'assets' to post converted pdf-file - %s", e
+            )
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=f"Failed request to 'assets' to post converted pdf-file",
+            ) from e
+        return request_to_post_assets.json()[0]["id"]
+
+    def upload_output_pdf_to_s3(self):
+        with tempfile.TemporaryDirectory() as tmp_dirname:
+            pdf_path = tmp_dirname / Path("badgerdoc_render.pdf")
```

Comment:
May you move `badgerdoc_render.pdf` to constant?

Reply:
moved to class variable